projects/fsastorefrontlib/src/occ/adapters/product-search/occ-product-search-page-normalizer.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import {
+  ConverterService,
+  Facet,
+  Occ,
+  OccProductSearchPageNormalizer,
+  PRODUCT_NORMALIZER,
+  ProductSearchPage,
+} from '@spartacus/core';
+
+@Injectable({ providedIn: 'root' })
+export class FSOccProductSearchPageNormalizer extends OccProductSearchPageNormalizer {
+  constructor(private converter: ConverterService) {
+    super(converter);
+  }
+
+  /**
+   * Overrides Spartacus filtering facet values
+   */
+  convert(
+    source: Occ.ProductSearchPage,
+    target: ProductSearchPage = {}
+  ): ProductSearchPage {
+    target = {
+      ...target,
+      ...(source as any),
+    };
+    if (source.products) {
+      target.products = source.products.map(product =>
+        this.converter.convert(product, PRODUCT_NORMALIZER)
```

Comment:
so now, we are returning plain product search results from the backend, no filtering or whatsoever on the frontend side?

Reply:
We still have this product filtering (which is also responsible for adding images) but I deleted methods for filtering facets.